src/main/java/seedu/address/model/InfoHandler.java

```diff
@@ -0,0 +1,198 @@
+    }
+
+    /**
+     * Retrieves the total amount of people who are currently quarantined.
+     *
+     * @return the number of people quarantined currently.
+     */
+    public int getTotalQuarantined() {
+        updateModelPersonList(PREDICATE_SHOW_ALL_QUARANTINED);
+        int totalQuarantined = getPersonList().size();
+        updateModelPersonList(PREDICATE_SHOW_ALL_PERSONS);
+
+        return totalQuarantined;
+    }
+
+    //=========== Get percentages ==================================================================
+
+    /**
+     * Gets the percentage of people who are currently infected.
+     *
+     * @return the percentage of infected people.
+     */
+    public double getInfectedOverPeople() {
+        int totalInfected = getTotalInfected();
+        int totalPeople = getTotalPeople();
+        // Check if dividing by zero
+        if (totalPeople == 0) {
+            return INVALID_RATIO;
+        }
+        return (double) totalInfected / totalPeople;
+    }
+
+    /**
+     * Gets the percentage of people who are currently quarantined.
+     *
+     * @return the percentage of quarantined people.
+     */
+    public double getQuarantinedOverPeople() {
+        int totalQuarantined = getTotalQuarantined();
+        int totalPeople = getTotalPeople();
+        // Check if dividing by zero
+        if (totalPeople == 0) {
+            return INVALID_RATIO;
+        }
+        return (double) totalQuarantined / totalPeople;
+    }
+
+    //============ Util ===========================================================================
+
+    /**
+     * Converts the given ratio into a percentage String.
+     * Returns {@code INVALID_PERCENTAGE} if given ratio is -1.0
+     * @param ratio
+     * @return
+     */
+    public String getRatioAsPercentage(double ratio) {
+        if (ratio == INVALID_RATIO) {
+            return INVALID_PERCENTAGE_STRING;
+        }
+        double percentage = HUNDRED_IN_DOUBLE * ratio;
+        return String.format(PERCENTAGE_FORMAT + "%%", percentage);
+    }
+
+    //============ Summary ========================================================================
+
+    /**
+     * Gets a summary of the current statistics.
+     */
+    public String getStatistics() {
+        return String.format(
+            TOTAL_PEOPLE_HEADER + " " + COUNT_FORMAT + "\n"
+            + TOTAL_LOCATIONS_HEADER + " " + COUNT_FORMAT + "\n"
+            + TOTAL_VISITS_HEADER + " " + COUNT_FORMAT + "\n"
+            + TOTAL_INFECTED_HEADER + " " + COUNT_FORMAT + "\n"
+            + TOTAL_QUARANTINED_HEADER + " " + COUNT_FORMAT + "\n"
+            + PERCENTAGE_INFECTED + " %s" + "\n"
+            + PERCENTAGE_QUARANTINED + " %s" + "\n",
+            getTotalPeople(), getTotalLocations(), getTotalVisits(),
+            getTotalInfected(), getTotalQuarantined(), getRatioAsPercentage(getInfectedOverPeople()),
+            getRatioAsPercentage(getQuarantinedOverPeople())
+        );
+    }
```

Comment:
It appears that you have added some new functionality here regarding some statistics. Can these be used already?

Reply:
Yes, these methods should already be functional barring unforeseen bugs.